lib/reports/backlog_report.rb

```diff
@@ -56,19 +56,14 @@ def self.generate_csv(start_date, end_date, exclude_duplicates = true)
       end
```

Comment:
It's not related to your change but in overall report backlog:
Line 42-47: Going through all the translations for each language and each day is definitely slow. We may need to preprocess the translations once and use it later